webapp/src/containers/createIncident/createIncidentMenu.tsx

```diff
@@ -21,6 +21,7 @@ const CreateIncidentPostMenuAction = ({postId}: PropTypes) => {
     const {pluginState} = usePluginApi();
     const dispatch = useDispatch();
     const post = useSelector((state: GlobalState) => getPost(state, postId));
+    const {SiteURL} = useSelector((state: GlobalState) => state.entities.general.config);
```

Comment:
Can we put this in a separate function to avoid duplicating this logic?

```suggestion
    const siteURL = useSelector(getSiteURL);
```